src/disable-amp.js

```diff
@@ -29,7 +29,7 @@ function preventAmp() {
         el[expando] = true;
 
         const url = el.getAttribute('data-amp-cur');
-        el.setAttribute('href', url);
+        el.setAttribute('ping', url);
```

Comment:
Because ping is the element that is called when the right button is clicked if you want to open a link on a new tab (at least in my case).

Can you confirm the same behaviour in your case?

Reply:
<b>Edit:</b>Sorry for that, I've checked again the element and appear to be "href".